server/main.lua

```diff
@@ -755,7 +734,7 @@ RegisterNetEvent('police:server:JailPlayer', function(playerId, time)
         ["date"] = currentDate
     })
     TriggerClientEvent("police:client:SendToJail", OtherPlayer.PlayerData.source, time)
-    TriggerClientEvent('QBCore:Notify', src, Lang:t("info.sent_jail_for", {time = time}))
+    TriggerClientEvent('ox_lib', src, {description = Lang:t("info.sent_jail_for", {time = time})})
```

Comment:
```suggestion
    TriggerClientEvent('ox_lib:notify', src, {description = Lang:t("info.sent_jail_for", {time = time})})
```
Looks like we missed one more

Reply:
Right, I'm just gonna push the version I have, because those actually have the changes ;p